rkt/api_service.go

```diff
@@ -390,6 +397,25 @@ func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest) {
 	return pod, manifest
 }
 
+func waitForMachinedRegistration(uuid string) error {
+	conn, err := dbus.SystemBus()
+	if err != nil {
+		return err
+	}
+	machined := conn.Object("org.freedesktop.machine1", "/org/freedesktop/machine1")
+	machineName := "rkt-" + uuid
+
+	var o dbus.ObjectPath
+	for i := 0; i < 10; i++ {
```

Comment:
I just remembered that machined has D-Bus signals "MachineNew". So we might be able to wait for the signal without this active loop. OTOH, it might not be as easy with godbus.